andela/bp-esa-frontend
# 164598240 Implement stat cards

#### What does this PR do?
This PR implements statistics cards to display visual automation status for Automation, Onboarding, Offboarding, Slack, Email and Freckle in the ESA dashboard page.

#### Description of Task to be completed?
- Create StatsCard component
- Add CSS styles for the component
- Write the test for the component
- Implement redux to fetch stats data from the backend APIs.

#### How should this be manually tested?
1. Clone the repo `https://github.com/andela/bp-esa-frontend`
2. Check out to `bp-esa-frontend` folder and run this command on your terminal `git fetch && git checkout `ft-statistics-cards-164598240` and run `yarn install` on your terminal
3. Start the application with the command `yarn start`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#164598240](https://www.pivotaltracker.com/story/show/164598240)

#### Postman Documentation
N/A

#### Screenshots (If applicable)
<img width="1666" alt="Screenshot 2019-03-18 at 11 17 21" src="https://user-images.githubusercontent.com/24792513/54516275-784cb000-496f-11e9-959a-e9e71cb8d5ab.png">
